NWN.Anvil/src/main/API/Object/NwAreaOfEffect.cs

```diff
@@ -18,6 +18,22 @@ internal NwAreaOfEffect(CNWSAreaOfEffectObject areaOfEffectObject) : base(areaOf
       AreaOfEffect = areaOfEffectObject;
     }
 
+    /// <summary>
+    /// Gets the spell from which the Area Of Effect was created.
+    /// </summary>
+    public NwSpell Spell 
+    {
+      get => NwSpell.FromSpellId((int)AreaOfEffect.m_nSpellId);
+    }
+
+    /// <summary>
+    /// Gets the Area Of Effect duration.
+    /// </summary>
+    public TimeSpan RemainingDuration
+    {
+      get => TimeSpan.FromMilliseconds(AreaOfEffect.m_nDuration);
+    }
+
```

Comment:
```suggestion
    /// <summary>
    /// Gets the spell from which the Area Of Effect was created.
    /// </summary>
    public NwSpell Spell => NwSpell.FromSpellId((int)AreaOfEffect.m_nSpellId);

    /// <summary>
    /// Gets the Area Of Effect duration.
    /// </summary>
    public TimeSpan RemainingDuration => TimeSpan.FromMilliseconds(AreaOfEffect.m_nDuration);
```

Minor formatting nits, otherwise LGTM :)